app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookFragmentTest.kt

```diff
@@ -71,6 +74,8 @@ class RoadBookFragmentTest {
     fun toRoadBookFragment() {
         // Ensure "use RoadBook preferences" is disabled
         PreferencesSetting.setRoadBookPrefs(testRule)
+        val courier = UsersPlaceHolder.USER_COURIER
+        GeneralUtils.fillUserEntryAndEnterTheApp(courier.email, courier.password)
```

Comment:
Not sure of what you are suggesting. The login is in the @before routine to be connected when we reach the roadbook fragment. Could you explain again ?

Reply:
Sorry now the 80 to 84, if you see the whole @Before function, you will remark that the last lines are still bypassing the login.
And this can be deleted as now you enter properly with the login.